src/net/sourceforge/kolmafia/utilities/FileUtilities.java

```diff
@@ -209,13 +214,23 @@ private static HttpResponse<InputStream> getResponseFromRequest(
       RequestLogger.trace("Requesting: " + remote);
     }
 
-    var client = getClient();
     HttpResponse<InputStream> response;
-    try {
-      response = client.send(request, BodyHandlers.ofInputStream());
-    } catch (IOException | InterruptedException e) {
-      StaticEntity.printStackTrace(e);
-      return null;
+    if (testingWithFakeData) {
```

Comment:
don't change the source code for the tests -- instead, make a `resetClient`, same as `GenericRequest` has, that you call in your tests when you want future calls to use the fake client instead of a real client, after mocking the fake client

Reply:
I'm not understanding the suggestion well enough to implement.  Agree that changing the core code is not the best thing.  What is your proposed resetClient a method of?  Maybe that will get me started.  Thanks.